MovingCameraForegroundEstimetor/MathematicalModels.py

```diff
@@ -251,6 +260,16 @@ def init(self):
         self.spatial_property = np.zeros((self.model_height * self.block_size, self.model_width * self.block_size),
                                          dtype=np.float32)
 
+    def update(self, var_init, var_trim, age_trim, theta_s, theta_d, dynamic, calc_probs, sensitivity, suppress):
```

Comment:
this 4: num_models, model_height, model_width, block_size don't need to get updated as well ?